src/vs/editor/browser/widget/diffEditorWidget2/diffEditorEditors.ts

```diff
@@ -149,7 +151,10 @@ export class DiffEditorEditors extends Disposable {
 	}
 
 	private _updateAriaLabel(ariaLabel: string | undefined): string | undefined {
-		const ariaNavigationTip = localize('diff-aria-navigation-tip', ' use Shift + F7 to navigate changes');
+		if (!ariaLabel) {
+			ariaLabel = '';
+		}
+		const ariaNavigationTip = localize('diff-aria-navigation-tip', ' use {0} to open the accessibility help.', this._keybindingService.lookupKeybinding('editor.action.accessibilityHelp')?.getAriaLabel());
```

Comment:
I think you want to check here if `accessibility.verbosity.diffEditor` is enabled?

Also, I'm not hearing it via the screen reader when I open a diff editor. are you?

Reply:
@hediet not sure if you missed these two remarks

Reply:
It works when the diff editor is open and you change the setting.
This has been an issue in the old diff editor as well, I have to look into this seperately.